<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformDialogue</class>
 <widget class="QDialog" name="TransformDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Rotation</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>134</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="yRot">
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="xRot">
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Yaw:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Roll:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="zRot">
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Map Rotation</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>134</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="mapXRot">
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Yaw:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="mapYRot">
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="mapZRot">
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Translation</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>128</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="xTr">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="yTr">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="zTr">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Map Translation</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>128</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="mapXTr">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="mapYTr">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="mapZTr">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Scale</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_6">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="xScl">
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="yScl">
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="zScl">
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Map Scale</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_7">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="mapXScl">
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="mapYScl">
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="mapZScl">
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TransformDialogue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TransformDialogue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xRot</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRot</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zRot</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapXRot</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapYRot</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapZRot</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xTr</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yTr</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zTr</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapXTr</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapYTr</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapZTr</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xScl</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yScl</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zScl</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapXScl</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapYScl</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapZScl</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TransformDialogue</receiver>
   <slot>updateDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateDocument()</slot>
 </slots>
</ui>
